replica-identity-full-performance.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: "On the performance of REPLICA IDENTITY FULL in Postgres"
-description: "The blog post analyses the performance impact of REPLICA IDENTITY FULL by looking at the source code and performing benchmarks."
+description: "Analyzing the performance impact of REPLICA IDENTITY FULL by looking at the source code and performing benchmarks."
```

Comment:
```suggestion
description: "How to analyze replica performance by looking at your source code and performing benchmarks."
```

Reply:
I meant the Postgres source code, actually. Let me clarify that.

Reply:
REPLICA IDENTITY FULL is the setting name in Postgres. It doesn't have to be capitalized, but I think most people will recognize it like that, because that's how you would typically set it:

```
ALTER TABLE foo REPLICA IDENTITY FULL;
```